user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/SampleInformationMain.java

```diff
@@ -196,21 +203,71 @@ private void downloadSampleMetadata() {
         .map(ProjectOverview::projectCode).orElseThrow();
     var experimentName = experimentInformationService.find(projectId.value(), experimentId)
         .map(Experiment::getName).orElseThrow();
-    downloadComponent.trigger(new WorkbookDownloadStreamProvider() {
-      @Override
-      public String getFilename() {
-        return FileNameFormatter.formatWithTimestampedContext(LocalDate.now(), projectCode,
-            experimentName,
-            "sample information",
-            "xlsx");
-      }
 
-      @Override
-      public Workbook getWorkbook() {
-        return templateService.sampleBatchInformationXLSXTemplate(projectId.value(),
-            experimentId.value());
-      }
-    });
+    asyncProjectService.sampleInformationTemplate(projectId.value(), experimentId.value(),
+            OPEN_XML)
+        .doOnSubscribe(showInProgress("preparing sample information"))
+        .doOnSuccess(resource -> handleSuccess(projectCode, experimentName).accept(resource))
+        .doOnError(this::handleError)
+        .doFinally(this::closePendingTaskNotification)
```

Comment:
I am a bit unseasy because the toast handling can be UI specific. Shouldn't we capture the UI in question and forward it to the single methods? What happens if the user navigates while a task is in progress?

Reply:
Is this covered by `@UiScope`? Do the toasts close with navigation or do they stay open?

Reply:
I understand your concern, however i think it is beyond the scope of this PR and consider this as an own investigation / issue, since it concerns the notification subsystem as a hole.